cnnimg.py

```diff
@@ -31,10 +31,7 @@
 import Image as img
 import numpy as np
 import os.path
-import warnings
 
-#Ignore warnings
```

Comment:
Yeah, the warnings are trivial, however, there could be an option to display them in the debug mode which could be set explicitly. Does it makes sense? Will you be able to add this feature?